.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-npm test
+./node_modules/.bin/npm-run-all preCommit linters test
```

Comment:
`preCommit` and `linters` don't exist in this repos scripts? 

https://github.com/AlaskaAirlines/WC-Generator/blob/3a9b8996afb6509e92d58869ad024e33bbc5a813/package.json#L45-L51